src/mappers/world/world_mapper.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include <map>
+#include <string>
+#include <vector>
+
+#include "src/mappers/country/country_mapper.h"
+#include "src/mappers/culture/culture_graphics_mapper.h"
+#include "src/mappers/provinces/province_mapper.h"
+#include "src/mappers/provinces/province_mapper_importer.h"
+#include "src/mappers/provinces/province_mapping_types.h"
+#include "src/mappers/technology/tech_mapping.h"
+#include "src/vic3_world/world/vic3_world.h"
+
+namespace mappers
+{
+/// <summary>
+/// Container class for all the other mappers.
+/// </summary>
+class WorldMapper
+{
+  public:
+   static WorldMapper LoadFromFiles(commonItems::ModFilesystem hoi4_mod_filesystem, const vic3::World& source_world);
+
+   WorldMapper(const CountryMapper&& country_mapper,
+       const ProvinceMapper&& provice_mapper,
+       const std::vector<mappers::TechMapping>&& tech_mapper,
+       const CultureGraphicsMapper culture_graphics_mapper);
+
+   const CountryMapper country_mapper;
+   const ProvinceMapper province_mapper;
+   const std::vector<mappers::TechMapping> tech_mapper;
+   const CultureGraphicsMapper culture_graphics_mapper;
```

Comment:
Look like these aren't being used yet. Maybe delete for now?

Reply:
Will be used in next patch, just keep for now.